src/Search/model/redirectParams.js

```diff
@@ -1,120 +1,35 @@
-import get from 'lodash/get';
-import { filterState } from '@folio/stripes/components';
 
-// 'filters' is a comma-separated list of 'FILTER.VALUE'
-//  e.g. 'available.Available online,source.Local,type.Books,type.eBooks'
-//
-// 'mapping' is an object specifying how to map filters:
-//  keys are 'name' elements of objects in filterConfig
-//  values may be:
-//   null (when the filter has no equivalent in the destination app and should be discarded)
-//   otherwise an object with keys:
-//    name: the name of the filter in the destination app
-//    values: may be either
-//     null (when the values are the same in the destination app)
-//     or a map of values in Codex search to corresponding values in destination app
-//
-function mapFilters(filters, mapping) {
-  const filterKeys = filterState(filters);
-  const acc = [];
 
-  Object.keys(filterKeys).forEach((fullName) => {
-    const [groupName, fieldName] = fullName.split('.');
-    const group = mapping[groupName];
-    if (group) {
-      if (!group.values) {
-        // The values do not need mapping
-        const mapped = `${group.name}.${fieldName}`;
-        acc.push(mapped);
-      } else {
-        const mappedVal = group.values[fieldName];
-        if (mappedVal) {
-          const mapped = `${group.name}.${mappedVal}`;
-          acc.push(mapped);
-        }
-      }
-    }
-  });
-
-  const res = acc.join(',');
-  // console.log(`mapped '${filters}' -> '${res}'`);
-  return res;
-}
-
-
-function redirectParamsKB(record, resources) {
+function redirectParamsKB(record) {
   const obj = {
-    _path: `/eholdings/titles/${record.id}`,
+    _path: `/inventory/view/${record.id}`,
```

Comment:
This looks like a typo. `redirectParamsLocal` handles links to inventory; shouldn't `redirectParamsKB` continue sending traffic to eHoldings?

Reply:
Oh, yes!, thank you Zak.